backend/src/graphql/resolvers/images/imagesLocal.ts

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable @typescript-eslint/require-await */
```

Comment:
will  we still support local images? You remove some parts of the corresponding code - the migration throws an error if you do not have an s3 configured.

Reply:
I was hesitating to remove the code before the migration is done. Maybe I will remove it straight away when I refactor (e.g. pass s3Client via context) so I don't have to maintain it.

Reply:
Ok, jsut for my understanding - its fine to keep the code for a moment for reference